opensrp-fp/src/main/java/org/smartregister/fp/common/job/ArchivedPostSterilizationJob.java

```diff
@@ -0,0 +1,52 @@
+package org.smartregister.fp.common.job;
+
+import androidx.annotation.NonNull;
+
+import org.joda.time.LocalDate;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.smartregister.fp.common.library.FPLibrary;
+import org.smartregister.fp.features.home.repository.PatientRepository;
+import org.smartregister.job.BaseJob;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import timber.log.Timber;
+
+public class ArchivedPostSterilizationJob extends BaseJob {
+
+    public static final String TAG = "ArchivedPostSterilizationJob";
+
+    @NonNull
+    @Override
+    protected Result onRunJob(@NonNull Params params) {
+        List<HashMap<String, String>> data = FPLibrary.getInstance().getPreviousContactRepository().getLatestSterilizeContacts();
+
+        try {
+            int count = 0;
+            LocalDate todayDate = LocalDate.now();
+            DateTimeFormatter pattern = DateTimeFormat.forPattern("dd-MM-yyyy");
```

Comment:
Please use constants instead of hard coded strings

Reply:
constant already created but that is in another PR once that PR will be merged then I will replace that